src/components/Panel/Panel.less

```diff
@@ -94,34 +135,22 @@
     border-right: 1px solid @ms-color-neutralLight;
     right: -40px;
```

Comment:
Please remove. This is moving the panel-main off the screen to the right by 40px.